src/garage/torch/algos/__init__.py

```diff
@@ -1,9 +1,9 @@
 """PyTorch algorithms."""
-from garage.torch.algos._utils import _Default  # noqa: F401
-from garage.torch.algos._utils import compute_advantages  # noqa: F401
-from garage.torch.algos._utils import filter_valids  # noqa: F401
-from garage.torch.algos._utils import make_optimizer  # noqa: F401
-from garage.torch.algos._utils import pad_to_last  # noqa: F401
+from garage.torch.algos._utils import _Default
+from garage.torch.algos._utils import compute_advantages
+from garage.torch.algos._utils import filter_valids
+from garage.torch.algos._utils import make_optimizer
```

Comment:
I think it's better if we put all of these in `garage.torch` rather than `garage.torch.algos`

Reply:
Moved `garage.torch.algos._utils` to `garage.torch._utils` and export `compute_advantages`, `filter_valids`, and `pad_to_last` to `garage.torch`.